src/liveEditor/listeners/params.ts

```diff
@@ -0,0 +1,14 @@
+import { VisualEditorCslpEventDetails } from "../types/liveEditor.types";
+
+interface EventListenerHandlerParams {
+    event: MouseEvent;
+    overlayWrapper: HTMLDivElement | null;
+    visualEditorContainer: HTMLDivElement | null;
+    previousSelectedEditableDOM: Element | null;
+    focusedToolbar: HTMLDivElement | null;
+    resizeObserver: ResizeObserver;
+    eventDetails: VisualEditorCslpEventDetails;
+    customCursor: HTMLDivElement | null;
+}
+
+export default EventListenerHandlerParams;
```

Comment:
We only have type in this file. Let's ensure we follow the structure where we keep all the types in *.types.ts format